src/trace/trace.cpp

```diff
@@ -56,16 +56,6 @@ constexpr pid_t Trace::METRIC_PID;
 
 std::string get_trace_name(std::string prefix = "")
 {
-    if (prefix.size() == 0)
-    {
-        prefix = nitro::env::get("LO2S_OUTPUT_TRACE");
-    }
-
-    if (prefix.size() == 0)
-    {
-        prefix = "lo2s_trace_{DATE}";
-    }
-
```

Comment:
Is it correct that this functionality should be removed? I don't see code that replaces this

Reply:
There is:
https://github.com/tud-zih-energy/lo2s/blob/issue-147-prt4-broken-options/src/config.cpp#L137